src/Coordinates.ts

```diff
@@ -0,0 +1,4 @@
+export type Coordinates = {
```

Comment:
Why did we add this? You didn't explain why in your commit msg.

Reply:
Added to commit mesage.